File: 이건희/level1/모의고사.py
def solution(answers):
    result = []
    score = [0, 0, 0]
    student_1 = [1, 2, 3, 4, 5]
    student_2 = [2, 1, 2, 3, 2, 4, 2, 5]
    student_3 = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]

    for i, answer in enumerate(answers):
        if answer == student_1[i % len(student_1)]:
            score[0] += 1
        if answer == student_2[i % len(student_2)]:
            score[1] += 1
        if answer == student_3[i % len(student_3)]:
            score[2] += 1

    for i, s in enumerate(score):
        if s == max(score):
            result.append(i + 1)

    return result

File: 김찬우/프로그래머스 /Lv.1/모의고사.java
import java.util.Arrays;
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(int[] answers) {
        int[] one = {1, 2, 3, 4, 5};
        int[] two = {2, 1, 2, 3, 2, 4, 2, 5};
        int[] three = {3, 3, 1, 1, 2, 2, 4, 4, 5, 5};
    
        int[] result = {0,0,0};
        
         List<Integer> answer = new ArrayList<Integer>();
        
        for(int i=0;i<answers.length;i++){
            
            if(answers[i]==one[i%5]){
                result[0]++;
            }
            if(answers[i]==two[i%8]){
                result[1]++;
            }
            if(answers[i]==three[i%10]){
                result[2]++;
            }
            
            
            
            
        }
        
        int max = Math.max(result[0], Math.max(result[1], result[2]));
            
            for(int j = 0; j<3;j++){
                
                if(result[j] == max){
                    answer.add(j+1);
                }
            }
       
        
        
        return answer.stream().mapToInt(Integer::intValue).toArray();
    }
}
